app/src/app/components/contents/downloads/Downloads.tsx

```diff
@@ -0,0 +1,67 @@
+import { useParams } from "react-router-dom";
+import { usePacketOutletContext } from "../../main/PacketOutlet";
+import { PacketHeader } from "../packets";
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from "../../Base/Accordion";
+import { Roles } from "../../../../types";
+import Artefacts from "./Artefacts";
+import OtherFiles from "./OtherFiles";
+import { Card, CardContent } from "../../Base/Card";
+import FileRow from "./FileRow";
+
+export default function Downloads() {
+  const { packetId, packetName } = useParams();
+  const { packet } = usePacketOutletContext();
+  const artefacts = packet?.custom?.orderly.artefacts;
+  const inputs = packet?.custom?.orderly.role
```

Comment:
sorry what exactly is inputs? maybe a more descriptive name if it refers to something else but I'm unsure

Reply:
It's my 'friendly name' for 'role' which is an even more opaque name - see https://github.com/mrc-ide/orderly2/blob/48cb8fba7fc9cf46a6d2187931064bb83756e4c4/inst/schema/orderly/orderly.json#L43

We could call this `inputFiles` or `resources` or `otherFiles`